actgen/evaluate_dqn.py

```diff
@@ -52,8 +52,8 @@ def parse_args(self):
                             help='Path to the result directory to save model files')
         parser.add_argument('--tag', type=str, default='default_exp',
                             help='A tag for the current experiment, used as a subdirectory name for saving models')
-        parser.add_argument('--gpu', default=False, action='store_true',
-                            help='Run training on GPU')
+        parser.add_argument('--disable_gpu', default=False, action='store_true',
+                            help='enforce training on CPU')
```

Comment:
If most of the argparse stuff is shared among all the scripts, maybe we should abstract that out at some point to make it easier to keep things up to date?